Nit and example

edburns opened the pull request:
On branch edburns-msft-function-nits Nit and example 
modified:   articles/azure-resource-manager/managed-applications/create-uidefinition-functions.md

- In this context, functions are invoked, not declared.

- Add example of coalesce() use from Quinn McHugh.

edburns committed:
On branch edburns-msft-function-nits In this context, functions are invoked, not declared.  Add example of coalesce() use from Quinn McHugh.
modified:   articles/azure-resource-manager/managed-applications/create-uidefinition-functions.md

tfitzmac commented on articles/azure-resource-manager/managed-applications/create-uidefinition-functions.md:
````diff
@@ -481,6 +481,48 @@ Assume `element1` and `element2` are undefined. The following example returns `"
 "[coalesce(steps('foo').element1, steps('foo').element2, 'foobar')]"
 ```
 
+   > [!NOTE]
````
This section is probably a little long for a Note. Since it is really a description of how and when to use the function, the whole section would work better outside of a Note.

edburns replied:
I agree with your observation.

tfitzmac commented on articles/azure-resource-manager/managed-applications/create-uidefinition-functions.md:
````diff
@@ -481,6 +481,48 @@ Assume `element1` and `element2` are undefined. The following example returns `"
 ```
 
+   > [!NOTE]
+   > This function is especially in the context of optional invocation that happens due to user action after the page loads.  For example, if the constraints placed on one field in the UI depend on the currently selected value of another, **initially non-visible** field.  In this case, `coalesce()` can be used to allow the function to be syntactically valid at page load time, while having the des ...
````
I think missing a word - "especially useful"

tfitzmac commented on articles/azure-resource-manager/managed-applications/create-uidefinition-functions.md:
````diff
@@ -481,6 +481,48 @@ Assume `element1` and `element2` are undefined. The following example returns `"
+   >     },
+   > ```
+   >
+   > If we want to condition the action of another field on the current
````
General guidance is to not use "we" so this part could revised: "To condition the action of another field on the current value of this field, use `coalesce()` ..."

edburns replied:
I agree with your modification.

edburns committed:
On branch edburns-msft-function-nits Apply suggestions from @tfitzmac
modified:   articles/azure-resource-manager/managed-applications/create-uidefinition-functions.md

ktoliver committed:
public repo edit
